doc/user/content/get-started.md

```diff
@@ -11,10 +11,8 @@ To help you get started with Materialize, we'll:
 - Explore its API
 - Set up a real-time stream to perform aggregations on Wikipedia user edits.
 
-{{< cta >}}
-
-<a href="https://www.katacoda.com/materialize/scenarios/intro-wikipedia" target="_blank">Click here to try the demo in your browser!</a>
-
+{{< cta href=/katacoda#intro-wikipedia target=_blank >}}
+Try the demo in your browser!
```

Comment:
I added the "click here" after noticing that 1 or 2 people scrolled right past the button without clicking.  We don't have a lot of traffic to this page though, so it's hard to be sure.

Reply:
Ack, restored. (Links that say "Click here" are a pet peeve of mine; I know the research says they convert better, but... ugh.)

Reply:
feel free to drop the "click here!"  that was meant to be an "fyi" comment, not an actual review

Reply:
👍